scripts/deploy-contracts.ts

```diff
@@ -140,25 +144,64 @@ export const deployContracts = async (config: Config): Promise<string | null> =>
         sequenceUtilsV1.address
       )
 
+      // SEQ-0001 patch
+      prompt.start("Deploying SEQ-0001 MainModuleUpgradableDuo v1\n")
+      const mainModuleUpgradableDuoV1 = await universalDeployer.deploy('MainModuleUpgradableDuo', getArtifactFactory(MAIN_MODULE_UPGRADABLE_DUO_V1), 0, txParams)
+      prompt.succeed("Deployed SEQ-0001 MainModuleUpgradableDuo v1: " + mainModuleUpgradableDuoV1.address + "\n")
+
+      prompt.start("Deploying SEQ-0001 MainModuleUpgradableDuo v2\n")
+      const mainModuleUpgradableDuoV2 = await singletonDeployer.deploy('MainModuleUpgradableDuo', getArtifactFactory(MAIN_MODULE_UPGRADABLE_DUO_V2), 0, txParams)
+      prompt.succeed("Deployed SEQ-0001 MainModuleUpgradableDuo v2: " + mainModuleUpgradableDuoV2.address + "\n")
+
+      prompt.start("Deploying migrator for SEQ-0001 v1\n")
+      const migratorToDuoV1 = await universalDeployer.deploy(
+        'MainModuleUpgradableDuoMigrator',
+        getArtifactFactory(MIGRATOR_TO_DUO_V1),
+        0,
+        txParams,
+        mainModuleUpgradableDuoV1.address,
+        "0x596aF90CecdBF9A768886E771178fd5561dD27Ab",
+        "0x5ca5d4cb6696df530c26b130a8fd86276a111f6696b3a8f2e76ff5edf94a2d84",
+        "0xc99c1ab359199e4dcbd4603e9b2956d5681241ceb286359cf6a647ca56e6e128"
+      )
+      prompt.succeed("Deployed migrator for SEQ-0001 v1: " + migratorToDuoV1.address + "\n")
+
+      prompt.start("Deploying migrator for SEQ-0001 v2\n")
+      const migratorToDuoV2 = await singletonDeployer.deploy(
+        'MainModuleUpgradableDuoMigrator',
+        getArtifactFactory(MIGRATOR_TO_DUO_V2),
+        0,
+        txParams,
+        mainModuleUpgradableDuoV2.address,
+        "0x761f5e29944D79d76656323F106CF2efBF5F09e9",
+        "0xacb659ac7f85fbbce197005235ced2d040c7b02942a9dfae647582393d5a4e83",
+        "0x6e2f52838722eda7d569b52db277d0d87d36991a6aa9b9657ef9d8f09b0c33f4"
+      )
+      prompt.succeed("Deployed migrator for SEQ-0001 v2: " + migratorToDuoV2.address + "\n")
+
       // v1 prod
 
       await deployGuard(
```

Comment:
Let's move the guard deployments outside the `skipWalletContext` branch.

Reply:
Good idea, but keeping it as-is because it depends on having the `Factory` `Contract` object to perform the guard deployment.